features/editable/cmake/layout_vs

```diff
@@ -0,0 +1,12 @@
+[build_folder]
```

Comment:
This file should match the documented one: https://github.com/conan-io/docs/pull/1567/files#diff-430fdd2a2f6acd2e29bda5773563cca4R135

Reply:
done

Reply:
Please, check comment here (https://github.com/conan-io/docs/pull/1567/files#r387704804). It'd be great if this file shows an example of jinja2 replacements (:+1: ) but also of "logic customized for a specific package".

Reply:
What do you mean? `{{settings.build_type}}` is Jinja and we are using `Release` as build type, which is indicates a logic behind.

Reply:
I've added a new file, layout_jinja, which has a logic there. The only difference between gcc and visual studio is the build folder.